src/inspect_evals/swe_bench/swe_bench.py

```diff
@@ -61,6 +62,10 @@ def swe_bench(
     """
     getLogger().handlers = []  # Swe-bench adds a global logger, which we disable.
 
+    assert find_spec(
+        "swebench"
+    ), "To run SWE-bench, please install the optional swebench depency, by running `pip install inspect-ai[swebench]`"
+
```

Comment:
Is this the right thing to tell users? In particular, this is right for people using the pypy version but wrong for devs and people pointing towards main (maybe thats fine though)

Reply:
Yeah, that's fine. Once this is on PyPI that will work well. For now though we might want to use a GitHub URL